docker-compose.yml

```diff
@@ -1,22 +1,67 @@
 version: '3'
  
 services:
- webdev:
-   environment:
-    DATABASE_URL: postgres://beluga:beluga@postgres:5432/beluga
-   build: .
-   ports:
-     - "8080:8080"
-   volumes:
-     - .:/app/
-   depends_on:
-     - postgres
-  
- postgres:
-   image: postgres
-   ports:
-     - "5432"
-   environment:
-     POSTGRES_USER: beluga
-     POSTGRES_PASSWORD: beluga
-     POSTGRES_DB: beluga
+  webdev:
```

Comment:
Are we going to be able to run all this on Heroku? API, worker, Redis, Postgres, seems like a lot 😀

Reply:
YUP 😄 

Fallback: URLs are all controlled by env variables and everything is in a container => put it somewhere else.